pages/advanced-algorithms/available-algorithms/leiden_community_detection.mdx

```diff
@@ -0,0 +1,160 @@
+It enhances the Louvain algorithm by addressing its limitations, particularly in situations where some identified communities are not well-connected. 
+This improvement is made by periodically subdividing communities into smaller, well-connected groups.
+With an $\mathcal{O}(Lm)$ runtime for $m$ edges and $L$ number of iterations, it suits large graphs.
+
+<Cards>
+  <Card
+    icon={<GitHub />}
+    title="Source code"
+    href="https://github.com/memgraph/mage/blob/main/cpp/leiden_community_detection_module/leiden_community_detection_module.cpp"
+  />
+</Cards>
+
+| Trait                    | Value                                                                                                       |
+| ------------------------ | --------------------- |
+| **Module type**          | algorithm             |
+| **Implementation**       | C++                   |
+| **Graph direction**      | undirected            |
+| **Relationship weights** | weighted / unweighted |
+| **Parallelism**          | parallel              |
+
+## Procedures
+
+<Callout type="info">
+You can execute this algorithm on [graph projections, subgraphs or portions of the graph](/advanced-algorithms/run-algorithms#run-procedures-on-subgraph).
+</Callout>
+
+### `get()`
+
+Computes graph communities using the Leiden algorithm.
+
+{<h4> Input: </h4>}
+
+- `subgraph: Graph` (**OPTIONAL**) ➡ A specific subgraph, which is an [object of type Graph](/advanced-algorithms/run-algorithms#run-procedures-on-subgraph) returned by the `project()` function, on which the algorithm is run.
+- `weight: string (default=null)` ➡ Specifies the default relationship weight. If not set, 
+  the algorithm uses the `weight` relationship attribute when present and otherwise 
+  treats the graph as unweighted.
+- `gamma: double (default=1.0)` ➡ Resolution parameter used when computing the modularity. Internally the value is divided by the number of relationships for an unweighted graph, or the sum of weights of all relationships otherwise.
+- `theta: double (default=0.01)` ➡ Controls the randomness while breaking a community into smaller ones.
+- `resolution_parameter: double (default=0.01)` ➡  Minimum change in modularity that must be achieved when merging nodes within the same community.
+- `max_iterations: int (default=inf)` ➡ Maximum number of iterations the algorithm will perform. If set to infinity, the algorithm will run until convergence is reached.
+
+{<h4> Output: </h4>}
+
+- `node: Vertex` ➡ Graph node.
+- `community_id: integer` ➡ Community ID. Defaults to $-1$ if the node does not belong to any community.
+- `communities: list` ➡ List of intermediate communities that a node has been part of across iterations.
+
+{<h4> Usage: </h4>}
+
+Use the following query to detect communities:
+
+```cypher
+CALL leiden_community_detection.get()
+YIELD node, community_id, communities;
+```
+
+### `get_subgraph()`
+
+Computes graph communities over a subgraph using the Louvain method.
```

Comment:
Why is Louvain on the Leiden page? I'm confused.

Reply:
mistake.